packages/ui/client/components/dashboard/TestFilesEntry.vue

```diff
@@ -44,12 +44,33 @@ import { filesFailed, filesSnapshotFailed, filesSuccess, time } from '../../comp
       </div>
     </template>
 
+    <template v-if="unhandledErrors.length">
+      <div i-carbon-checkmark-outline-error />
+      <div>
+        Errors
+      </div>
+      <div class="number" text-red5>
+        {{ unhandledErrors.length }}
+      </div>
+    </template>
+
     <div i-carbon-timer />
     <div>Time</div>
     <div class="number" data-testid="run-time">
       {{ time }}
     </div>
   </div>
+  <template v-if="unhandledErrors.length">
+    <div bg="red500/10" text="red500" p="x3 y2" max-w-xl m-2 rounded>
+      <h3 text-center mb-2>
+        Unhandled Errors
+      </h3>
+      <p text="sm" font-thin data-testid="unhandled-errors">
+        Vitest caught {{ unhandledErrors.length }} error{{ unhandledErrors.length > 1 ? 's' : '' }} during the test run.<br>
+        This might cause false positive tests. Resolve unhandled errors to make sure your tests are not affected.
```

Comment:
Shouldn't we display errors somehow? (A new tab? Hideable list?) This might be a bit more useful

Reply:
I've added some details about the errors. 
Took [this](https://github.com/vitest-dev/vitest/blob/main/packages/vitest/src/node/error.ts#L25) method as a starting point. I've kept things simple for now, for example it doesn't display the stack trace. Please let me know if you think I should add more info.

Also test cases do not cover every branch yet, as I was not yet able to come up with a sample test for all the edge cases.

Reply:
We should definitely print stack trace, maybe without the code block, but stack should be present

Reply:
Ok, will do

Reply:
@sheremet-va Added stack trace to errors. Updated the description with new screenshots.